packages/sit-onyx/src/components/OnyxNavBar/OnyxNavBar.stories.ts

```diff
@@ -65,30 +58,57 @@ export const Default = {
     logoUrl: "/onyx-logo.svg",
     appName: "App name",
     default: () => [
-      h(OnyxNavButton, { label: "Item 1", link: "/" }),
+      h(OnyxNavItem, { label: "Item 1", link: "/" }),
       h(
-        OnyxNavButton,
-        { label: "Item 2", link: "/test" },
+        OnyxNavItem,
+        { label: "Item 2" },
         {
           default: () => ["Item 2", h(OnyxBadge, { dot: true, color: "warning" })],
           children: () => [
-            h(OnyxNavItem, { label: "Nested item 2.1", link: "#" }),
-            h(OnyxNavItem, { label: "Nested item 2.2", link: "#", active: true }),
-            h(OnyxNavItem, { label: "Nested item 2.3", link: "https://onyx.schwarz" }),
+            h(
```

Comment:
Should we keep the default nav bar example minimal so the source code does not get too overwhelming?
We could add a new example "Nested items"